### PR TITLE
seo changes and sitemap for insights site

### PR DESCRIPTION
**Pull request by HussainKhanSherwani**
# Pull Request Template

Before submitting this PR, please make sure you have completed the following checklist. This helps ensure that our codebase remains high quality and that our features work well across all supported devices and themes.

### Code Review
- [ ] I have carefully reviewed my code and believe it is final from my side.
- [ ] I have ensured there is no hardcoded color in my changes. Colors should be defined in theme files and reused.

### Theme Compatibility
- [ ] I have tested the new feature or updates in both light and dark themes.
- [ ] I confirm that the visual aspects of the feature are compatible and appealing in both themes.

### Device Compatibility
- [ ] I have tested my changes on mobile screen sizes.
- [ ] I have tested my changes on tablet screen sizes.
- [ ] I have tested my changes on laptop screen sizes.

### Functional Testing
- [ ] I have thoroughly tested the functionality of my changes.
- [ ] I am confident that my changes do not negatively impact existing features.

### Testing Documentation
Please provide a brief description of the tests you have performed to verify your changes. Include any relevant scenarios or edge cases.

- Example test 1: Tested feature X by uploading the image
- Example test 2: Tested feature X by doing these things.

# Demo Video
Please provide a demo video of your changes. This helps reviewers understand the changes and verify the functionality.

You can record the video using tools like Loom(https://www.loom.com/), Clip by ClickUp(https://clickup.com/features/clips), Sendspark(https://www.sendspark.com/), etc.

Include the link to the video below and make sure the video covers
- [ ] Explain the task/bug you are solving
- [ ] Show the changes in the UI
- [ ] Show the changes in the codebase
- [ ] Show the changes you tested to make sure it works
- [ ] Show the changes in various screen sizes
- [ ] Show the changes in both light and dark themes

### CODE REVIEW

File: insights-ui/src/app/robots.txt/route.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import getBaseUrl from '@dodao/web-core/utils/api/getBaseURL';
```
Comment (Hassaan-Qaisar) [inline insights-ui/src/app/robots.txt/route.ts:1]: If the project only has a single static sitemap, you can indeed create a simple robots.txt file directly in the public folder.

File: insights-ui/src/app/crowd-funding/projects/[projectId]/page.tsx [hunk 2]
```diff
@@ -41,17 +40,11 @@ export async function generateMetadata({ params }: { params: Promise<{ projectId
     },
     openGraph: {
       title: `${data.projectDetails.name} - Crowdfunding Project | DoDAO`,
-      description: `Explore project details, funding status, and insights for "${data.projectDetails.name}".`,
+      description: `DoDAO Agentic Insights provides detailed analysis and insights for "${data.projectDetails.name}" with graphs, reports and metrics like growth,financial health ,traction, valuation ,Execution and speed.`,
```
Comment (Hassaan-Qaisar) [inline insights-ui/src/app/crowd-funding/projects/[projectId]/page.tsx:43]: Check HTML encoding or string conversion causing &quot; entities in meta description.
it is coming like this &quot;Olympian Motors&quot
